src/main/java/org/opensrp/repository/postgres/TaskRepositoryImpl.java

```diff
@@ -390,6 +391,10 @@ public Task updateTask(Task task) {
 		return get(task.getIdentifier());
 	}
 
+	@Override
```

Comment:
@githengi  This method signature was defined by Kigamba on Plan Evaluator, therefore to avoid compilation issues. I have thrown an exception on Not Implemented. Does this look okay?

Reply:
you can just implement it

Reply:
leave it as it is the method should be amended to add plan filter